synchronizer/ext_control.go

```diff
@@ -0,0 +1,112 @@
+package synchronizer
+
+import (
+	"bufio"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/0xPolygonHermez/zkevm-node/log"
+)
+
+const (
+	externalControlFilename = "/tmp/synchronizer_in"
```

Comment:
if this path must be created and filled with the commands, why don't we convert it in a config param?
I understand that if we runt the docker, first we should copy the file into the container. Am I right?

Reply:
in fact no, the code create the file and if exists skip all contents (is for live commands). I think that is not worth to allow configure it becuase is for debug/test propouses. If you want to use inside a docker you can map the volume of /tmp/ or the file by itself